src/wasm-features.h

```diff
@@ -40,21 +40,15 @@ struct FeatureSet {
     Multivalue = 1 << 9,
     GC = 1 << 10,
     Memory64 = 1 << 11,
-    // TODO: Remove this feature when the wasm spec stabilizes.
-    GCNNLocals = 1 << 12,
-    RelaxedSIMD = 1 << 13,
-    ExtendedConst = 1 << 14,
-    Strings = 1 << 15,
-    MultiMemory = 1 << 16,
+    RelaxedSIMD = 1 << 12,
+    ExtendedConst = 1 << 13,
+    Strings = 1 << 14,
+    MultiMemory = 1 << 15,
     MVP = None,
     // Keep in sync with llvm default features:
     // https://github.com/llvm/llvm-project/blob/c7576cb89d6c95f03968076e902d3adfd1996577/clang/lib/Basic/Targets/WebAssembly.cpp#L150-L153
     Default = SignExt | MutableGlobals,
-    // GCNNLocals are opt-in: merely asking for "All" does not apply them. To
-    // get all possible values use AllPossible. See setAll() below for more
-    // details.
-    All = ((1 << 17) - 1) & ~GCNNLocals,
-    AllPossible = (1 << 17) - 1,
+    All = (1 << 17) - 1,
```

Comment:
Shouldn't this be 16 now that you removed one?

Reply:
Yes 😢